SenderEmail/.env

```diff
@@ -0,0 +1,4 @@
+HOST=smtp.gmail.com
+PORT=587
+USERNAME=mostafadfrg@gmail.com
+PASSWORD=qzbcgpgalkqvipki
```

Comment:
You should delete this file from remote repository and add it to `.gitignore` file.